### ISSUE TITLE
Extension stopped working with no notable errors

### ISSUE THREAD

**Issue by creage** [madskristensen/SvgViewer#1]
Installed it yesterday, was working fine, and suddenly stopped today. SGVs are just not displayed. No errors. Tried both on VS2017, and VS2019, extension version 1.0.7. Reinstalling does not help either.

**Comment by madskristensen** [comment]
Can you please share the .svg file?

**Comment by creage** [comment]
Well, it does not work for any of them, but here you are

```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 51.2 (57519) - http://www.bohemiancoding.com/sketch -->
    <title>add-dashboard</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="add-dashboard" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="Group-2" transform="translate(13.000000, 11.000000)" fill-rule="nonzero">
            <g id="Group" stroke="#143C4D">
                <circle id="Oval-3" cx="4.5" cy="4.5" r="4.5"></circle>
            </g>
            <rect id="Rectangle-9" fill="#143C4D" x="4" y="2" width="1" height="5" rx="0.2"></rect>
            <rect id="Rectangle-9-Copy" fill="#143C4D" transform="translate(4.500000, 4.500000) rotate(-270.000000) translate(-4.500000, -4.500000) " x="4" y="2" width="1" height="5" rx="0.2"></rect>
        </g>
        <g id="Group" transform="translate(2.000000, 5.000000)" fill="#0D3C49" fill-rule="nonzero">
            <path d="M11,8 L12,8 C10.3333333,10.3333333 11,12.6666667 14,15 C13,15 12,15 11,15 L11,15 C10.4477153,15 10,14.5522847 10,14 L10,9 C10,8.44771525 10.4477153,8 11,8 Z" id="Rectangle-10"></path>
            <path d="M11,0 L18,0 C18.5522847,-1.01453063e-16 19,0.44771525 19,1 L19,6 C19,6.55228475 18.5522847,7 18,7 C18,7 18,7 18,7 C15.5,5 13.5,6.5 13,7 C12.3333333,7 11.6666667,7 11,7 L11,7 C10.4477153,7 10,6.55228475 10,6 L10,1 C10,0.44771525 10.4477153,1.01453063e-16 11,0 Z" id="Rectangle-10-Copy-2"></path>
            <rect id="Rectangle-10-Copy" x="0" y="8" width="9" height="7" rx="1"></rect>
            <rect id="Rectangle-10-Copy-3" x="0" y="0" width="9" height="7" rx="1"></rect>
        </g>
    </g>
</svg>
```

And, if it might help, here is my VS info
```
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.0.0 Preview 1.1
VisualStudio.16.Preview/16.0.0-pre.1.1+28408.50
Microsoft .NET Framework
Version 4.7.03190

Installed Version: Enterprise

ASP.NET and Web Tools 2017   16.0.01169.0
ASP.NET and Web Tools 2017

ASP.NET Web Frameworks and Tools 2017   5.2.61002.0
For additional information, visit https://www.asp.net/

Azure App Service Tools v3.0.0   16.0.01169.0
Azure App Service Tools v3.0.0

Azure Functions and Web Jobs Tools   16.0.01169.0
Azure Functions and Web Jobs Tools

C# Tools   2.11.0-beta1-63519-03+4c7d79a254288388d97d793c4087db6ebf00d2fd
C# components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Common Azure Tools   1.10
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

File Icons   2.7
Adds icons for files that are not recognized by Solution Explorer

Fix File Encoding   1.4.1
Prevents Visual Studio from adding BOM to UTF-8 files.

JetBrains ReSharper Ultimate 2018.3.1   Build 183.0.20181225.200351
JetBrains ReSharper Ultimate package for Microsoft Visual Studio. For more information about ReSharper Ultimate, visit http://www.jetbrains.com/resharper. Copyright © 2019 JetBrains, Inc.

Microsoft Azure Tools   2.9
Microsoft Azure Tools for Microsoft Visual Studio 0x10 - v2.9.11108.1

Microsoft Continuous Delivery Tools for Visual Studio   0.4
Simplifying the configuration of Azure DevOps pipelines from within the Visual Studio IDE.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual Studio Tools for Containers   1.1
Develop, run, validate your ASP.NET Core applications in the target environment. F5 your application directly into a container with debugging, or CTRL + F5 to edit & refresh your app without having to rebuild the container.

Node.js Tools   1.5.1114.1 Commit Hash:2413337bb6550a03bc79eda03f528c82ec9438fd
Adds support for developing and debugging Node.js apps in Visual Studio

NPM Task Runner   1.4.90
Adds support for npm scripts defined in package.json directly in Visual Studio's Task Runner Explorer. Includes full support for Yarn

NuGet Package Manager   4.6.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Open Command Line   2.1.218
Opens a command line at the root of the project. Support for all consoles such as CMD, PowerShell, Bash etc. Provides syntax highlighting, Intellisense and execution of .cmd and .bat files.

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

ResourcePackage Extension   1.0
ResourcePackage Visual Studio Extension Detailed Info

Snapshot Debugging Extension   1.0
Snapshot Debugging Visual Studio Extension Detailed Info

SQL Server Data Tools   16.0.61811.08220
Microsoft SQL Server Data Tools

SVG Viewer   1.0.7
Makes it much easier to work with SVG files by showing a live preview in the bottom-right corner of the code window

Syntax Highlighting Pack   2.8.134
Adds syntax highlighting and snippet support for a wide variety of programming languages such as Clojure, Go, Jade, Lua, Swift, Ruby and many more...

TypeScript Tools   16.0.1101.2001
TypeScript Tools for Microsoft Visual Studio

Visual Basic Tools   2.11.0-beta1-63519-03+4c7d79a254288388d97d793c4087db6ebf00d2fd
Visual Basic components used in the IDE. Depending on your project type and settings, a different version of the compiler may be used.

Visual F# Tools 10.3 for F# 4.5   16.0.0.0.  Commit Hash: 518bf573d151acbdf0c62d604477c841d08b4de3.
Microsoft Visual F# Tools 10.3 for F# 4.5

Visual Studio Code Debug Adapter Host Package   1.0
Interop layer for hosting Visual Studio Code debug adapters in Visual Studio

Visual Studio Tools for Containers   1.0
Visual Studio Tools for Containers

VisualLinter   1.1.0
Visual Studio JavaScript linter using ESLint.

VSColorOutput   2.6.3
Color output for build and debug windows - http://mike-ward.net/vscoloroutput
```
